packages/gatsby-theme-newrelic/index.js

```diff
@@ -6,8 +6,11 @@ export { default as HamburgerMenu } from './src/components/HamburgerMenu';
 export { default as Icon } from './src/components/Icon';
 export { default as MDXCodeBlock } from './src/components/MDXCodeBlock';
 export { default as NewRelicLogo } from './src/components/NewRelicLogo';
+export { default as Overlay } from './src/components/Overlay';
+export { default as Portal } from './src/components/Portal';
 export { default as SearchInput } from './src/components/SearchInput';
 export { default as Surface } from './src/components/Surface';
+export { default as SwiftSearch } from './src/components/SwiftSearch';
```

Comment:
Out of curiosity, are we thinking that sites may want to implement search outside of the global header search experience? If not, maybe we should hold off on exporting this.

Reply:
we weren't sure so we wanted to leave it flexible, but i'm fine with us delivering the search as part of the header